### PR TITLE
update insight release note

### PR DESCRIPTION
**Pull request by justedennnnn**

### CODE REVIEW

File: docs/zh/docs/insight/intro/releasenote.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+## 2024-08-31
+
+### v0.30.0
+
+#### Insight Server
+
+- **新增** 服务详情页增加关联容器的流量分布以及流量指标
+- **新增** 集群、节点负载监控仪表盘
+- **新增** 新增 Nvidia GPU 资源监控
+- **新增** 告警列表新增告警状态并支持过
```
Comment (samzong) [inline docs/zh/docs/insight/intro/releasenote.md:14]:
```suggestion
- **新增** 告警列表新增告警状态字段，并支持搜索过滤
```

File: docs/zh/docs/insight/intro/releasenote.md [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
+- **新增** 链路组件 Jaeger 与 OpenTelemetry Collector 增加内置的告警规则
```
Comment (samzong) [inline docs/zh/docs/insight/intro/releasenote.md:15]:
```suggestion
- **新增** 链路组件 `Jaeger` 与 `OpenTelemetry Collector` 增加内置的告警规则
```

保持一致

File: docs/zh/docs/insight/intro/releasenote.md [hunk 3]
```diff
@@ -2,6 +2,26 @@
...
+- **新增** 安装参数中增加 `日志模块`，`事件模块` 的功能开关，允许不启用日志模块
+- **优化** Jaeger Collector 默认的启动参数，优化组件性能
```
Comment (samzong) [inline docs/zh/docs/insight/intro/releasenote.md:17]:
```suggestion
- **优化** `Jaeger Collector` 默认的启动参数，优化组件性能
```

**Review by samzong — COMMENTED** [review]
👍

**Comment by windsonsea** [comment]
If ready, pls remove the `draft` status.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**